IdentityServer/Infrastructure/Data/UserIdentityDbContext.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace IdentityServer.Infrastructure.Data
+{
+    public class UserIdentityDbContext : IdentityDbContext<Core.Users.UserIdentity, IdentityRole<int>, int>
+    {
+        public UserIdentityDbContext(DbContextOptions options)
+            : base(options)
+        {
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            var user = modelBuilder.Entity<Core.Users.UserIdentity>()
+                .ToTable("User");
+            user.HasKey(u => u.Id);
```

Comment:
Do you need that here? Convention of ef probably doesn't require specifying key directly.

Reply:
That's a remnant of trying to fix a bug, nice catch.